.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I believe we usually don't commit IDE settings. Not a blocker, tho.

Reply:
Fine to include since Lisa mentioned:
> A [.vscode/settings.json](https://github.com/.vscode/settings.json) file associates [Makefile.devhelpers](https://github.com/Makefile.devhelpers) as a Makefile.